layout/_macro/post.swig

```diff
@@ -190,30 +197,26 @@
       {% endif %}
 
       {% if is_index %}
-        {% if post.description %}
-          {{ post.description }}
-          <div class="post-more-link text-center">
-            <a class="btn" href="{{ url_for(post.path) }}">
-              {{ __('post.read_more') }} &raquo;
-            </a>
-          </div>
-        {% elif post.excerpt  %}
+        {% if post.excerpt  %}
```

Comment:
In previous releases, I use the `description` front -matter as a way to specify post excerpt. As a result, removing this code might confuse users that uses this feature. So my thought is to reserve this feature.